app/services/rapid_ready_for_decision/constants.rb

```diff
@@ -48,6 +48,9 @@ class Constants
 
     DISABILITIES_BY_CODE = DISABILITIES.map { |k, v| [v[:code], k] }.to_h
 
+    # prostate cancer (1), sleep apnea (1), rhinitis (1), sinusitus (4)
+    MAS_DISABILITIES = [7528, 6847, 6522, 6510, 6511, 6512, 6513, 6514].freeze
+
```

Comment:
> Should MAS go into the constants class?

This is fine. It would be better if you grouped the DCs maybe on separate lines:
```suggestion
    MAS_DISABILITIES = [
      7528, # prostate cancer
      6847, # sleep apnea
      6522, # rhinitis
      6510, 6511, 6512, 6513, 6514 #sinusitus
    ].freeze
```